packages/calypso-e2e/src/jest-playwright-config/global-setup.ts

```diff
@@ -13,29 +13,33 @@ export default async (): Promise< void > => {
 		return;
 	}
 
-	if ( AUTHENTICATE_ACCOUNTS.length > 0 ) {
-		const browser = await chromium.launch( {
-			...pwConfig.launchOptions,
-			headless: true,
-		} );
-
-		await Promise.all(
-			AUTHENTICATE_ACCOUNTS.map( async ( accountName ) => {
-				const testAccount = new TestAccount( accountName );
-				if ( await testAccount.hasFreshAuthCookies() ) {
-					return;
-				}
-
-				const page = await browser.newPage( pwConfig.contextOptions );
-				page.setDefaultTimeout( envVariables.TIMEOUT );
-
-				await testAccount.logInViaLoginPage( page );
-				await testAccount.saveAuthCookies( page.context() );
-
-				await page.close();
-			} )
-		);
-
-		await browser.close();
+	// If the list of accounts for which to pre-authenticate and save cookies
+	// for is empty, then don't run.
+	if ( AUTHENTICATE_ACCOUNTS.length <= 0 ) {
```

Comment:
Changing the conditional to reject early.

Reply:
nit: since array length cannot be negative:

```suggestion
	if ( AUTHENTICATE_ACCOUNTS.length === 0 ) {
```

Reply:
Merged changes, thanks for the nit.